roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -173,21 +173,9 @@ def required_images(self):
             if self.get_var("osm_use_cockpit", default=True, convert=bool):
                 required.add(self._registry_console_image(image_tag, image_info))
 
-        # images for containerized components
-        def add_var_or_default_img(var_name, comp_name):
-            """Returns: default image from comp_name, overridden by var_name in task_vars"""
-            default = "{}/{}:{}".format(image_info["namespace"], comp_name, image_tag)
-            required.add(self.template_var(self.get_var(var_name, default=default)))
-
-        if self.get_var("openshift_is_containerized", convert=bool):
+        if self.get_var("openshift_is_atomic", convert=bool):
             if 'oo_nodes_to_config' in host_groups:
-                add_var_or_default_img("osn_image", image_info["name"] + "-node")
-            if 'oo_masters_to_config' in host_groups:  # name is "origin" or "ose"
```

Comment:
This statement is removed due to the fact that openshift_is_containerized is not longer set; this would have only been checked on atomic, not to mention doesn't match how we currently do image strings.